enterprise/app/filter/filter.tsx

```diff
@@ -369,6 +394,11 @@ export default class FilterComponent extends React.Component<FilterProps, State>
                 <Wrench /> {commandValue}
               </span>
             )}
+            {(minimumDurationValue || maximumDurationValue) && (
+              <span className="advanced-badge">
+                <Timer /> {minimumDurationValue} - {maximumDurationValue}
```

Comment:
nit: the timer icon is not very legible since it's very small (12px wide). Could you increase it to 16px?

![image](https://user-images.githubusercontent.com/2414826/210678864-e3b8b473-3719-4fff-8a41-516bee6beffe.png)

I think the relevant style is `.advanced-badge svg` which is setting it to 12px in `filter.css`.

Reply:
Do you want me to make them all bigger? It's the same size as the other ones right now.

Reply:
Actually, I think we could switch to the `Clock` icon which we already use for invocation durations:

![image](https://user-images.githubusercontent.com/2414826/211035425-57dac3a0-add8-426f-bc1f-0337b90e189e.png)

The clock hands are larger compared to the `Timer` icon so it should be legible at 12px.